crypto/src/test/java/org/web3j/crypto/MnemonicUtilsTest.java

```diff
@@ -44,7 +41,12 @@
      */
     @Parameters
     public static Collection<Object[]> data() throws IOException {
-        String data = Files.lines(Paths.get(SAMPLE_FILE)).collect(Collectors.joining("\n"));
+        BufferedReader br = new BufferedReader(new FileReader(SAMPLE_FILE));
```

Comment:
This should also be a utility function, but in the Files utility class.

Also, please can you ensure you have a space after the `for (` for consistency.